ascend-sdk/src/main/java/ai/evolv/ascend/android/AscendAction.java

```diff
@@ -0,0 +1,11 @@
+package ai.evolv.ascend.android;
+
+public interface AscendAction<T> {
```

Comment:
Should this not be something like ValueUpdateListener? What is "Action" in this context? Also, "Ascend" is redundant with the namespace.